flink-runtime-web/web-dashboard/src/app/pages/job/status/job-status.component.html

```diff
@@ -51,7 +51,7 @@ <h2>{{ jobDetail.name }}</h2>
   <div class="operate">
     <span *ngIf="statusTips">{{ statusTips }}</span>
     <ng-container *ngIf="!statusTips">
-      <a nz-popconfirm nzTitle="Cancel Job?" (nzOnConfirm)="cancelJob()" *ngIf="webCancelEnabled && (jobDetail.state=='RUNNING' || jobDetail.state=='CREATED' || jobDetail.state=='RESTARTING')">Cancel Job</a>
+        <a nz-popconfirm nzTitle="Cancel Job?" nzOkText="Yes" nzCancelText="No" (nzOnConfirm)="cancelJob()" *ngIf="jobDetail.state=='RUNNING' || jobDetail.state=='CREATED' || jobDetail.state=='RESTARTING'">Cancel Job</a>
```

Comment:
- please revert the indentation change
- why are you removing the webCancelEnabled check?

Reply:
Hi! Thank you for your comment. I fixed.